pytorch_lightning/trainer/training_loop.py

```diff
@@ -518,12 +479,15 @@ def optimizer_step(self, optimizer, opt_idx, batch_idx, train_step_and_backward_
     def on_before_zero_grad(self, optimizer):
         self.trainer.call_hook('on_before_zero_grad', optimizer)
 
+    def optimizer_zero_grad(self, batch_idx, optimizer, opt_idx):
```

Comment:
Not sure the accelerator needs to perform zero_grad.

Reply:
@awaelchli ^^

Reply:
+1